src/style/content/conversation/input-bar.less

```diff
@@ -52,8 +52,8 @@
   min-height: @conversation-input-line-height;
   max-height: @conversation-input-max-height;
   flex: 1 1;
-  margin-top: (@conversation-input-min-height - @conversation-input-line-height) / 2;
-  margin-bottom: (@conversation-input-min-height - @conversation-input-line-height) / 2;
+  margin-top: calc((@conversation-input-min-height - @conversation-input-line-height) / 2);
+  margin-bottom: calc((@conversation-input-min-height - @conversation-input-line-height) / 2);
```

Comment:
why do we need to use `calc` now? Is that a dynamic computation now?

Reply:
I'd say so, it wouldn't work because of the em values

Reply:
OOhh right, the `conversation-input-line-height` is now in `em`, right 👌 

It's impossible to see on the PR cause this variable is affected to another variable value, thus my question